app/views/admin/participants/_details.html.erb

```diff
@@ -0,0 +1,58 @@
+<table class="govuk-table">
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">Full name</th>
+      <td class="govuk-table__cell"><%= latest_induction_record.user.full_name %></td>
+      <td class="govuk-table__cell">
+        <%= govuk_link_to edit_name_admin_participant_path(latest_induction_record.participant_profile) do %>
+          Change <span class="govuk-visually-hidden">name</span>
+        <% end %>
+      </td>
+    </tr>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">Email address</th>
+      <td class="govuk-table__cell"><%= latest_induction_record.user.email %></td>
+      <td class="govuk-table__cell">
+        <%= govuk_link_to edit_email_admin_participant_path(latest_induction_record.participant_profile) do %>
+          Change <span class="govuk-visually-hidden">email</span>
+        <% end %>
+      </td>
+    </tr>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">ID</th>
+      <td class="govuk-table__cell"><%= latest_induction_record.user.id %></td>
+      <td class="govuk-table__cell"></td>
+    </tr>
```

Comment:
```suggestion
    <tr class="govuk-table__row">
      <th scope="row" class="govuk-table__header">ID</th>
      <td class="govuk-table__cell"><%= latest_induction_record.user.id %></td>
      <td class="govuk-table__cell"></td>
    </tr>
     <tr class="govuk-table__row">
      <th scope="row" class="govuk-table__header">External ID</th>
      <td class="govuk-table__cell"><%= latest_induction_record.preferred_identity.external_identifier %></td>
      <td class="govuk-table__cell"></td>
    </tr>
```

This might be useful so that we support can see what ID a lead provider might see. What do you think?

Reply:
Yeah, i think that's a good idea. Reduces potential confusion (hopefully) between LP and admin